docs/learn/learn-proxies.md

```diff
@@ -105,6 +105,13 @@ multisig would be created.
 
 ## How to set up a Proxy
 
+> NOTE: The first proxy you add should always be of _type_ `Any`.
```

Comment:
This warning applies only to Anonymous proxies, so it should go under `Creating Anonymous Proxies`.

Reply:
Updated